### PR TITLE
🌚  Council activities

### PR DESCRIPTION
**Pull request by WRadoslaw**
@jodator @p-sad Hey guys, could you take a quick look and activities we decided to put into council feed? If our picks are enough than PR can be converted from draft.

### CODE REVIEW

File: packages/ui/src/council/components/Activities/NewCandidateContent.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { CouncilRoutes } from '@/council/constants'
+import { useCandidateIdByMember } from '@/council/hooks/useCandidateIdByMember'
+import { NewCandidateActivity } from '@/council/types/CouncilActivities'
+
+export const NewCandidateContent: ActivityContentComponent<NewCandidateActivity> = ({ activity }) => {
+  const { candidateId } = useCandidateIdByMember(activity.memberId)
+  return (
+    <ActivityRouterLink to={`${CouncilRoutes.council}/${candidateId}`}>
+      {activity.candidateHandle} has applied to council election.
+    </ActivityRouterLink>
```
Comment (p-sad) [inline packages/ui/src/council/components/Activities/NewCandidateContent.tsx:14]: This link leads nowhere - it should probably lead to the candidate's profile in current election.
```suggestion
    <ActivityRouterLink to={`${CouncilRoutes.currentElection}?candidate=${candidateId}`}>
      {activity.candidateHandle} has applied to council election.
    </ActivityRouterLink>
```

File: packages/ui/src/council/components/Activities/CandidacyWithdrawContent.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { ActivityContentComponent } from '@/common/components/Activities/ActivityContent'
+import { CandidacyWithdrawActivity } from '@/council/types/CouncilActivities'
+
+export const CandidacyWithdrawContent: ActivityContentComponent<CandidacyWithdrawActivity> = ({ activity }) => (
+  <>Council candidate {activity.candidateHandle} has resigned.</>
```
Comment (p-sad) [inline packages/ui/src/council/components/Activities/CandidacyWithdrawContent.tsx:7]: I'd put the candidate handle inside of a `MemberModalLink` component like in other activities that display a member. I also considered linking to their candidacy, but I guess that's pretty much pointless given that they've resigned.

**Review by p-sad — COMMENTED** [review]
Looks good for the most part, I only have suggestions for some of the activity components.

**Review by jodator — COMMENTED** [review]
Looks good :)

One thing is that probably some designs for the icons might be needed. Would you be able to prepare a story book for the council activities? This might be the easiest way to share what we'd need.

**Pull request merged**